modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -1990,6 +2058,21 @@ private ObjectEntry _addOrUpdateObjectEntry(
 			ServiceContextTestUtil.getServiceContext());
 	}
 
+	private void _addOrUpdateSystemObjectExtendedProperties(
```

Comment:
I don't think it's intuitive to assert something inside this method. During the whole PR review I thought this was just adding or updating. I think we should rename this or move the assert out.

Reply:
I'll move the assert out

Reply:
Done!

Reply:
Thanks 🙏🏻